src/gmusic-theme.js

```diff
@@ -101,6 +105,16 @@ window.GMusicTheme = class GMusicTheme {
           parent.appendChild((new DOMParser()).parseFromString(normalSVG, 'text/xml').firstChild);
         }
       }
+
+      // DEV: Google sometimes changes its logo byitself, we need to monitor this
+      this.logoObserver = new MutationObserver(() => {
+        this._drawLogo();
+      });
+      this.logoObserver.observe(parent, {
+        childList: true,
+        attributes: true,
+        subtree: true,
+      });
```

Comment:
I have hesitation about using a `MutationObserver` since it could be not too great at performance. Would be nicer if we maybe hid Google's logo and replaced ours via CSS?